src/core/tracking.js

```diff
@@ -105,8 +105,23 @@ export function getAction(type) {
   }[type] || TRACKING_TYPE_INVALID;
 }
 
+export function doNotTrackEnabled({
```

Comment:
Good call. The tracking code logs whether or not GA is on but this is separate and worth logging ("Why is `trackingEnabled` set to `true` but I have no logging?!")